File: src/main/java/com/tao/Hello.java
package com.tao;

public class Hello {

    public static void main(String[] args) {
     person p=new person("yuan",66.5f, 1.7f);
     p.hello();
     //p.weight=66.5f;
     //p.height=1.7f;
     System.out.println(p.bmi());
     System.out.println(p.name+"  "+p.height+"  "+p.weight);
     System.out.println("中文系統");
     System.out.print("請輸入英文成績");

     /*Integer a1=8;
     float a3=34.5f;*/
    }
}

File: src/main/kotlin/com/tao/Hello.kt
package com.tao

fun main(args: Array<String>) {

   var h=Human(weight=66.5f,height=1.7f)

    h.hello()
    h.bmi()
}
class Human(var weight :Float, var height :Float){

    fun bmi() {

     var bmi=weight/(height*height)
     //return bmi
        println(bmi)

    }
    fun hello(){
        println("Hello Kotlin")
    }
}
